model/src/main/proto/survey.proto

```diff
@@ -0,0 +1,204 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Corresponds to a single survey.
+message Survey {
+  // The ID of the survey. This is a string value that uniquely identifies the survey.
+  // The ID is guaranteed to be unique across all surveys globally for the purpose of analysis.
```

Comment:
```suggestion
  // The ID should ostensibly be unique across all surveys globally for the purpose of analysis.
```

We don't actually make guarantees since that would require a remote service like Firestore keeping track of used IDs. GUIDs are statistically unlikely to overlap, but that isn't the same as a guarantee.

Reply:
Thanks, this makes sense. I can see why we really can't make the guarantee when thousands of devices are involved.